lib/restaurant.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'customer'
+
+class Restaurant
+  attr_accessor = :menu
+
+  def initialize
+    @menu = { 'Burger' => '£10',
```

Comment:
Good, although you're hardcoding your list of dishes into the Restaurant class. Can you think of another way of doing this?